components/general/ClearFilters.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default class ClearFilters extends React.Component{
```

Comment:
svg -> do pliku svg. Tutaj ew. ClearFIltersButton który to osadza i handluje onClick